extensions/wikia/ContentReview/controllers/SpecialContentReviewController.class.php

```diff
@@ -0,0 +1,34 @@
+<?php
+use  Wikia\ContentReview\Models\ReviewModel;
+
+class SpecialContentReviewController extends WikiaSpecialPageController {
+
+	public static $status = [
+		ReviewModel::CONTENT_REVIEW_STATUS_UNREVIEWED => 'content-review-status-unreviewed',
+		ReviewModel::CONTENT_REVIEW_STATUS_IN_REVIEW => 'content-review-status-in-review',
+		ReviewModel::CONTENT_REVIEW_STATUS_APPROVED => 'content-review-status-approved',
+		ReviewModel::CONTENT_REVIEW_STATUS_REJECTED => 'content-review-status-rejected'
+	];
+
+	function __construct() {
+		parent::__construct( 'ContentReview', 'ContentReview', true );
+	}
+
+	public function index() {
+		$this->wg->Out->setPageTitle( wfMessage( 'content-review-special-title' )->escaped() );
```

Comment:
In a special page controller, `$this->getOutput()` is preferred over `$this->wg->Out`. Also, it's good to consider just making use of `$this->specialPage->setHeaders();` instead which also sets the correct robot policy for special pages.